packages/trader/src/App/Components/Routes/binary-link.jsx

```diff
@@ -16,11 +16,19 @@ const BinaryLink = ({ active_class, to, children, ...props }) => {
     }
 
     return to ? (
-        <NavLink to={path} activeClassName={active_class || 'active'} exact={route.exact} {...props}>
+        <NavLink
+            data-testid='dt_nav_link'
```

Comment:
since this NavLink is using in binary-link component its better to make this data-testid something similar to the component. `dt_nav_link` is so common